Farmacy/ViewController/LoginViewController.m

```diff
@@ -30,6 +32,49 @@ - (void)viewDidLoad {
     
 }
 
+//For who don't want to login with Facebook account
+- (IBAction)loginUserWithParse:(id)sender {
+    NSString *username = self.usernameTextField.text;
+    NSString *password = self.passwordTextField.text;
+    
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        if (error != nil) {
+            NSLog(@"User log in failed: %@", error.localizedDescription);
```

Comment:
nit: should we display an error if login fails?

Reply:
I've updated with Login Alert Message to alert users and allow them to log in again